src/app/utils/steemApi.js

```diff
@@ -0,0 +1,11 @@
+import { api } from '@steemit/steem-js';
+
+import stateCleaner from 'app/redux/stateCleaner';
+
```

Comment:
It might be nice to wrap `getAccount` in the state cleaner too, such that when a GDPR user attempts to login it also fails ... because I imagine this will be one method a user might check that we have satisfied their request?